### ISSUE TITLE
Add an option to display more information about contact suggested via extension Contacts suggestions (LDAP)

### ISSUE THREAD

**Issue by ceskyDJ** [the-djmaze/snappymail#1723]
**Is your feature request related to a problem? Please describe.**

We have a public faculty LDAP at FIT BUT, which can be used for contact suggestions even in Snappymail. There is a catch, since there are a few people working/studing, you need to distinguish between them. Sometimes a student has exactly the same full name as a professor for example, so it could be really bad if you choose badly. Of course you can use email addresses in this example (students email addresses uses special subdomain) but when there are two employees with the same surname, you want to see departments, where they work, as you can safely diferenciate them without additional googling. So, it's not tragedy you see just a full name and email address of suggested people but it could be very helpful be able to see some additional information (e.g. department in this case). I think this isn't just my problem but could be helpful in larger companies, too.

**Describe the solution you'd like**
I hope the easy and good working solution would be to add something like "Additional information" to the configuration, where you specify LDAP class(es) you want to add to the autocomplete items, and show these classes' values in brackets after the email address in the autocomplete.

So, e.g.  an answer for "lu", when using class `organizationalUnit`:
- "Luke Numbers" <luke@numbers.name> (Accountant)
- "Lucy Nice" <me@lucy-nice.eu> (Secretary of CEO)

**Comment by the-djmaze** [comment]
An email address does allow a comment, between `(` and `)` characters at the end.
But the current code doesn't handle them.

This has to ve changed to get your proposal implemented.

**Comment by ceskyDJ** [comment]
I know that email address (in e.g. From header) doesn't allow this. I just speak about the list of suggested items in autocomplete — show brackets at the end of shown suggested items and insert ONLY full name and email address to the address (e.g. From) input, so brackets and its content will be trimmed.

**Comment by the-djmaze** [comment]
Yes i know there is a difference regarding RFC's and what an SMTP accepts.
That's why comments are currently not supported, but RFC 822 does.

**Comment by ceskyDJ** [comment]
I think we speak about something else. Let me describe it a little more deeply.

I don't want to change the address in (e.g.) From header at all. I just need to modify items in autocomplete result list.

Let me show it in a picture for better understanding. The autocomple should look like this (sorry for ugly image):
![image](https://github.com/user-attachments/assets/aeffee4d-bb56-43e6-a256-9f6e1ba365b3)

And after I select the first item:
![image](https://github.com/user-attachments/assets/cec54f65-2fc4-4cf2-aaaf-2b07598a85a2)

I need the comment just for the searching process (for my information who the person really, so I can safely select the correct recepient of my email).
